src/components/CategoryOverview/CategoryOverview.js

```diff
@@ -21,14 +20,6 @@ const classnames = require('classnames/bind').bind(style);
 const CategoryOverview = (props) => {
   const isContextCategory = props.category.id === CATEGORY_IDS.CONTEXT;
 
-  const indicators = props.category.overviewIndicatorIds.map((indicatorId) => {
-    const value = props.city
-      ? props.city.indicators[indicatorId]
-      : getMinAndMaxForIndicator(indicatorId, props.cities);
-
-    return { id: indicatorId, value };
-  });
-
```

Comment:
Previously data was aggregated before being sent down to the card. Now the 'range card' handles it.